File: SWIFT/Cap03/c03ex02.swift
// c03ex02.swift

import Foundation

var R: Double, A: Double

print("Entre valor do raio: ", terminator: "")
if let inputR = readLine(), let doubleR = Double(inputR) {
    R = doubleR
} else {
    fatalError("Entrada invalida para o raio")
}

while getchar() != "\n" { }

A = Double.pi * pow(R, 2)

print("Resultado = \(String(format: "%.2f", A))")

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap03/c03ex02.kt
// c03ex02.kt

fun main() {
    var r: Double
    var a: Double

    print("Entre valor do raio: ")
    r = readLine()?.toDouble() ?: 0.0

    a = Math.PI * Math.pow(r, 2.0)

    println("Resultado = %.2f".format(a))

    println("\nTecle <Enter> para encerrar... ")
    readLine()
}
